python/CMakeLists.txt

```diff
@@ -60,12 +60,23 @@ declare_mlir_python_extension(TorchMLIRPythonExtensions.Main
 # Optionally handle JIT IR importer.
 ################################################################################
 
+option(LIBTORCH_SRC_BUILD "Build libtorch from source" OFF)
+option(LIBTORCH_CACHE "Cache libtorch downloads" OFF)
```

Comment:
Happy to discuss the best default. IMHO we should defer to always build with latest (and if latest is broken we fix it) since that is the assumption right now - we live on top of master pytorch.  If we enabling caching we will let the user take the burden of unsetting it when they hit issues like the one on discord. For the naive rebuild -- we trade off a libtorch download for the sake of no stale local build. So I would suggest we leave this OFF for now and then once other loose ends are tied we can explore flipping it.

Reply:
I hear you. I see both sides. I think the situation I was looking at is that you're working on a bug, and re-compiling introduces a new libtorch without you rebasing or anything. Perhaps the right answer is just to be really vocal about the setting in the dev docs and loud in the build process.